app/assets/stylesheets/application.scss

```diff
@@ -46,6 +46,17 @@ body {
   .container {
     padding-top: 20px;
   }
+  table {
+    width: 350px;
+    text-align: center;
+  }
+  .form {
+    color: #000;
+    h1 {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3